cloudbuild/run-tests.sh

```diff
@@ -93,6 +99,7 @@ run_tests() {
     --test_output=errors \
     --noshow_progress \
     --noshow_loading_progress \
+    --keep_going \
```

Comment:
Will this impact presubmit?

Reply:
No, I think I could even remove this and it wouldn't change anything.